lib/octokit/default.rb

```diff
@@ -137,6 +137,12 @@ def proxy
         ENV['OCTOKIT_PROXY']
       end
 
+      # Default SSL verify mode from ENV
+      # @return [String]
+      def ssl_verify_mode
+        ENV['OCTOKIT_SSL_VERIFY_MODE']
```

Comment:
So because most folks will not have this set, it will default to false which is a breaking change.

See https://github.com/lostisland/faraday/blob/923f90558c22326c9a757a7ff9dfb3a006d0abc8/lib/faraday/adapter/net_http.rb#L124-L130 for the implementation details

What do you think about doing something like this instead?

```ruby
ENV.fetch("OCTOKIT_SSL_VERIFY_MODE") { true }
```

Reply:
That makes sense, totally forgot about defaults.